### ISSUE TITLE
Empty VertAlign node not deleted, causes Excel to bail

### ISSUE THREAD

**Issue by richardtallent** [EPPlusSoftware/EPPlus#116]
https://github.com/EPPlusSoftware/EPPlus/blob/7c6392004716c8ec820b52a279764a8a609adf9a/src/EPPlus/Style/ExcelRichText.cs#L226

I fixed this in my personal fork of 4.5.3.3 with this line:

```C#
DeleteNode(VERT_ALIGN_PATH.Substring(0, VERT_ALIGN_PATH.LastIndexOf("/")));
```

**Comment by JanKallman** [comment]
Thanks, I will apply this fix shortly

**Comment by JanKallman** [comment]
Fixed in Version 5.2
